### DISCUSSION TITLE
Open question regarding Machine Identity in Kratos

### DISCUSSION THREAD

**Discussion by lbdremy** [ory/kratos#1922] [Q&A] [answered]
Hello,

I'm wondering what do you mean in the documentation by Machine identity here https://www.ory.sh/kratos/docs/admin/managing-users-identities/#creating-a-machine-identity ?

Will this identity be identity attached to Identity Credentials called API Key or something similar? 

Thanks for your vision on this topic

**Comment by vinckr** [comment]
"Machine Identities" are what we call Service Accounts, so basically bots. 

>Will this identity be identity attached to Identity Credentials called API Key

Something like this is the basic idea, but I could not find the respective issue. Maybe @zepatrik knows if one already exists?

**Comment by davidspiess** [comment]
I was wondering about this too.
I read about this in https://github.com/ory/kratos/issues/1106, but it seemed like the implementation of Personal Access Tokens in Kratos was rejected.

**Comment by aeneasr** [comment, marked as answer]
The best way to achieve this today is probably using Ory Hydra with OAuth2 Client Credentials. We will add more credential types in the future to Ory Kratos but as of today (as linked in the docs) this isn't possible yet really.
